CHANGELOG.md

```diff
@@ -1,18 +1,23 @@
 # Changelog
 
+### Version 1.0.4
+- NEW: Added .eslintrc and .babelrc
```

Comment:
Take a looks at the new [Contribution Guidelines for writing consistent commit messages](https://github.com/ahmadawais/WPGulp/blob/master/CONTRIBUTING.md) for this repo.